spec/challenge_spec.rb

```diff
@@ -17,8 +20,6 @@
 
   describe '#each' do
     it 'stores minimal state' do
-      require 'binding_of_caller'
```

Comment:
Similar to the comment about rearranging the body of a class, it was just a habit of placing `require*`s at the top of a file.